pkg/api/metrics.go

```diff
@@ -22,10 +22,12 @@ type Metrics struct {
 	ReceivedMetadata      prometheus.Counter
 	FailedSamples         prometheus.Counter
 	FailedMetadata        prometheus.Counter
-	SentSamples           prometheus.Counter
+	IngestedSamples       prometheus.Counter
 	SentMetadata          prometheus.Counter
 	SentBatchDuration     prometheus.Histogram
```

Comment:
Why was "SentSamples" changed to "IngestedSamples" but "SentMetadata" and "SentBatchDuration" not changed?

Reply:
That is changed in order to make metric names equal to column names in the instance information table